src/components/cms/SkillCardScrollList/SkillCard.js

```diff
@@ -74,27 +74,22 @@ class SkillCard extends Component {
   };
 
   componentDidMount = () => {
+    window.addEventListener('resize', () => {
+      this.updateWindowDimensions();
+      this.updateBtnsDisplay();
+    });
     this.updateWindowDimensions();
   };
 
   updateWindowDimensions = () => {
     let scrollCards = 1;
-    switch (true) {
-      case window.innerWidth >= 1680:
-        scrollCards = 5;
-        break;
-      case window.innerWidth >= 1400:
-        scrollCards = 4;
-        break;
-      case window.innerWidth >= 1120:
-        scrollCards = 3;
-        break;
-      case window.innerWidth >= 840:
-        scrollCards = 2;
+    // Correct positioning of Right arrow for SkillCards upto 20 is supported with this for loop.
+    const MAX_CARDS_SUPPORT = 20;
+    for (let i = MAX_CARDS_SUPPORT; i > 1; i--) {
+      if (window.innerWidth >= window.innerWidth * 0.15 + 19 + i * 280) {
```

Comment:
Can you add appropriate comments to explain the formula?

Reply:
@akshatnitd Comments explaining formula added.